scripts/envs/chatbot

```diff
@@ -11,15 +11,15 @@ export APP_RUN_COMMAND="streamlit run chatbot_ui.py"
 # https://github.com/redhat-ai-dev/developer-images/tree/main/models/granite-3.1-8b-instruct-gguf
 export INIT_CONTAINER="quay.io/redhat-ai-dev/granite-3.1-8b-instruct-gguf:latest"
 
-# https://github.com/redhat-ai-dev/developer-images/tree/main/model-servers/llamacpp_python/0.3.8
-export MODEL_SERVICE_CONTAINER="quay.io/redhat-ai-dev/llamacpp_python:0.3.8"
+# https://github.com/redhat-ai-dev/developer-images/tree/main/model-servers/llamacpp_python/0.3.16
+export MODEL_SERVICE_CONTAINER="quay.io/redhat-ai-dev/llamacpp_python:0.3.16"
```

Comment:
no tag in quay repo https://quay.io/repository/redhat-ai-dev/llamacpp_python?tab=tags